### PR TITLE
Release: 0.2.0

### PR DESCRIPTION
**Pull request by treyburn**
## [0.2.0]

- Increase supported custom integration icon size from 32x32 to 256x256
- Loosen restrictions on MAC address input validation, permitting all types that the runZero platform can accept.
- Remove ability to specify last seen timestamp on asset imports. Though the server retained the timestamp as a
  custom property, it is not a merge-able property. This is likely to cause confusion. API users are advised
  to use any invented custom property to hold this value.
- Added new 'RateLimitError' exception to inform the user when they reach their daily usage limit.
- Rename all occurrences in API docs, models, types, routes of 'sources' or 'custom sources' with 'custom integrations'

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0]
+
+- Increase supported custom integration icon size from 32x32 to 256x256
```
Comment (deathbots) [inline CHANGELOG.md:10]: The rate limiting changelog entry

**Review by deathbots — DISMISSED** [review]
Tiny thing is all I yet see

File: CHANGELOG.md (hunk 1 above)
Comment (treyburn) [reply CHANGELOG.md:10]: Ah I guess we missed that. How does this sound: `Added new 'RateLimitError' type to inform the users when they reach their daily usage limit` ?

**Review by zbartl — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (deathbots) [reply CHANGELOG.md:10]: Sounds perfect

**Review by deathbots — APPROVED** [review]

**Pull request merged**